modules/replicator/src/main/java/org/apache/ignite/internal/replicator/listener/ReplicaListener.java

```diff
@@ -19,11 +19,8 @@
 
 import java.util.concurrent.CompletableFuture;
 import org.apache.ignite.internal.replicator.message.ReplicaRequest;
-import org.apache.ignite.network.ClusterNode;
 
-/**
- * Replica listener.
- */
+/** Replica listener. */
```

Comment:
Why did you reformat completely valid comment? Please stop making stylistic changes in unrelated pieces of code.

Reply:
We discussed it as a team and decided not to remove the changes.